torchaudio/_internal/module_utils.py

```diff
@@ -33,3 +34,14 @@ def wrapped(*args, **kwargs):
                 raise RuntimeError(f'{func.__module__}.{func.__name__} requires {req}')
             return wrapped
     return decorator
+
+
+def depricate(direction: str):
```

Comment:
Does pytorch already have something like this? I imagine we'd want to match their format.

Also, nit, it's [deprecate](https://en.wikipedia.org/wiki/Deprecation).